### DISCUSSION TITLE
My impression from new update

### DISCUSSION THREAD

**Discussion by odvigajlo** [niels747/2D-Weather-Sandbox#40] [General]
If to be short I'M REALLY IMPRESSED!
But now in details:
Vertical Resolution: i remember i suggested adding it before, and now it added! So i can say one thing IT'S IMPRESSIVE! Storms became REALLY detailed with it!
New Visuals: They look really cool! They look almost same as real life! When i first run simulator i got that feeling of mid summer!
Velosity Vector Display: i can't say anything because i don't know how to access it 😳
Trees Changes: Just love them!

Also want to mention: My FPS got 2.5x bigger in update! Even Though there are no optimization in changelog

UPD: After playing with new vertical resolution more: STORMS ARE MCUH BETTER NOW! Previously i would need to wait a few game days to see small anvils, while now I NEED 1 GAME DAY TO SEE IT! I'M REALLY IMPRESSED! Also shelf clouds became much better and common!

**Comment by Meso22** [comment]
Yeah the new update is sick as!

**Reply by odvigajlo** [reply]
Yeah

**Comment by GRolLevente** [comment]
@odvigajlo Press TAB for vector view. The vector view will only be visible at low zoom levels. In the temperature view, the vectors are displayed at all zoom levels.

**Reply by odvigajlo** [reply]
Thanks you!

**Reply by niels747** [reply]
Now it's only displayed only when zoomed in, for both the temperature and realistic display mode.

**Comment by MET905** [comment]
Hey guys, im not sure if im doing something wrong but I noticed my storms are moving very slowly and die off very quickly. the wind tool does not work so im not sure how to get things moving... i even turned off the atmospheric drag and still no change. but i will agree THIS NEW UPDATE IS BAD ASS!

**Reply by Meso22** [reply]
Could you send a pic of all of your settings?

**Reply by MET905** [reply]
Shared them below.

**Reply by Meso22** [reply]
sick I'll take a look

**Comment by MET905** [comment]
![1](https://user-images.githubusercontent.com/113155608/209412280-eeda85e6-9bda-426d-ad14-e9f01c886877.png)
![2](https://user-images.githubusercontent.com/113155608/209412281-61ae4feb-9e55-4724-bb9d-bb23919b1b9c.png)
Sorry about that my Images never Loaded in lol
im pretty good with meteorology but could you explain to me what the precip threshold and groth rate does id really appreciate it. im starting to think those settings are the reason for my weak storm systems

**Reply by Meso22** [reply]
Try changing the water weight back down to the default 0.5 and see if that helps also send me a picture of the water vapor and temperature display modes.

**Reply by MET905** [reply]
![1](https://user-images.githubusercontent.com/113155608/209413915-48c679f3-fcea-4dcc-9525-945f16984229.png)
![2](https://user-images.githubusercontent.com/113155608/209413917-1bb0b838-b795-41ad-ac79-2a76cf571c05.png)
Thanks for helping me out, I hope we can fix this I have not really had the chance to play with the new update because f this issue.
im pretty good with meteorology but could you explain to me what the precip threshold and groth rate does id really appreciate it. im starting to think those settings are the reason for my weak storm systems

**Reply by Meso22** [reply]
Your precip threshold and growth rate are at the default setting so they shouldn't be the problem. I have no idea what the precip threshold and growth rate do because I am not the dev. But I do think the problem is lack of moisture and heat because it looks like you have a low amount of heat around 14-20c (57-68 Fahrenheit) and not very high moisture. And also what size is your maps horizontal res and vertical res.

**Reply by MET905** [reply]
your right on the money! I even add a warn front and the storms are pathetic lol! I may have to add another area of water and make it as the golf of Mexico to get some moisture into the atmosphere. and my map is maxed out. Horizontal and Vertical. I may make another map and lower the map

**Reply by Meso22** [reply]
Your better off having all the resolutions at the highest level for extra realism.

**Reply by MET905** [reply]
Yeah, I just need to fix this Moisture issue. it makes sense... if the storm doesn't have enough water vapour to keep going its obviously going to fizzle out as the front moves.

**Reply by Meso22** [reply]
It is also good to not always rely on front's to form thunderstorms sometimes you can set the wind to full and the drag to 0.1 and wait for things to heat up and just let convection do it's thing

**Reply by MET905** [reply]
oh no no, I'm trying to start a blizzard lol thunderstorm development is easy haha!

**Reply by Meso22** [reply]
![image](https://user-images.githubusercontent.com/105577122/209415109-8db5c51d-7287-42f0-b9b1-9848f33958cc.png)
This is a very large thunderstorm that I managed to create doing exactly that

**Reply by Meso22** [reply]
Oh!! I thought you wanted massive supercells! haha!😂

**Reply by MET905** [reply]
I should have said at the beginning. my bad lol! 
Im going to build a new map

**Reply by Meso22** [reply]
I live in Australia so every time somebody says storm I just think of the average supercells we get during spring and summer!

**Reply by Meso22** [reply]
Yeah I have tried to make blizzards and it is very difficult

**Reply by MET905** [reply]
Haha!! gotcha! im from Ontario Canada im trying to recreate the blizzard we are getting now lol

**Reply by MET905** [reply]
yeah I think the issue is the water temperatures creating a lake shadow of warm air so snow turns into rain.

**Reply by Meso22** [reply]
yeah I found a way to combat that issue

**Reply by MET905** [reply]
please share you fix for that issue. my lake effect snow events end in rain every time lol!!!

**Reply by Meso22** [reply]
I just usually play with the global heating settings and set the starting height to the surface so that I can cool things down much quicker.

**Reply by MET905** [reply]
![1](https://user-images.githubusercontent.com/113155608/209416835-94b46bce-671e-4ca7-a940-186f853de434.png)
This should create some Interesting systems! All that moisture from the golf rising up the mountains should create some crazy stuff.  also added lake Erie for lake effect precip

**Reply by Meso22** [reply]
now that's a cool map!

**Reply by niels747** [reply]
Hi, nice to see what you guys are doing with the sim! I would like to answer the questions about some of the settings.
Precipitation Threshold +°C: the minimum cloudwater density at non-freezing temperatures before precipitation can form.
Precipitation Threshold -°C: the minimum cloudwater density at below freezing temperatures before precipitation can form.
The growth rates control how fast droplets grow. These are also different for positive and negative temperatures, because that is how it is in real life. I tuned the standard values to be realistic for a vertical resolution of 300, at lower res other values might give better results.

**Reply by niels747** [reply]
Of Courseit is way more complicated in real life.
Here is some of the sceince involved: http://pressbooks-dev.oer.hawaii.edu/atmo/chapter/chapter-7-precipitation-processes/

**Reply by MET905** [reply]
Thank you Niels!

**Comment by MET905** [comment]
To be more helpful i even added a Skew T fro ya. thank you for helping me fellow weather enthusiast
![2](https://user-images.githubusercontent.com/113155608/209412625-fabdbbb8-adbb-4cc5-a9c9-ffb54b3d3234.png)

**Reply by wilburcoding** [reply]
try adding a little bit of wind. Higher evaporation rates with higher water temperatures also help.

**Reply by niels747** [reply]
That is a really stable atmosphere, if the green line is left of the red line, the clouds won't be able to rise.

**Comment by MET905** [comment]
Wanted to wish everyone a Merry Christmas and a happy New year!

**Reply by Meso22** [reply]
Merry Christmas!
